webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -93,7 +93,8 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
       label: 'Durations',
     },
     {
-      info: `${humanReadableFloat(bestValidation)} (${experiment.config.searcher.metric})`,
+      info: bestValidation && `${humanReadableFloat(bestValidation)}` +
+      `(${experiment.config.searcher.metric})`,
```

Comment:
**nit:** I think this strips out a space between the readable float and the metric type in parentheses. How about:
```
info: bestValidation && `
  ${humanReadableFloat(bestValidation)} (${experiment.config.searcher.metric})
`,
```

Reply:
IMO this looks better 
```
      info: bestValidation && `${humanReadableFloat(bestValidation)} ` +
      `(${experiment.config.searcher.metric})`,
```
but either works

Reply:
or the same one you have with the ` on the same line